docs/getting_started.rst

```diff
@@ -344,7 +344,7 @@ The Client Credential grant is suitable for machine-to-machine authentication. Y
 
 Point your browser to http://127.0.0.1:8000/o/applications/register/ lets create an application.
 
-Fill the form as show in the screenshot below, and before saving take note of ``Client id`` and ``Client secret`` we will use it in a minute.
+Fill the form as show in the screenshot below, and before saving, carefully take note of ``Client id`` and ``Client secret``. Since version 2 the ``Client secret`` will be encrypted. We will use it in a minute.
 
 .. image:: _images/application-register-client-credential.png
```

Comment:
Since you are fixing this documentation, please update the screenshot to match the current release of the code. This version has a prompt message that tries to remind the users to copy the secret before saving.

See attached example (feel free to just use it!).

![application-register-client-credential](https://user-images.githubusercontent.com/4356471/171913379-b6cec180-42e3-4ceb-b337-160156861f3f.png)

Thanks!